hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/SparkJob.groovy

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.linkedin.gradle.hadoopdsl.job;
+
+/**
+ * Job class for type=spark jobs.
+ * <p>
+ * In the DSL, a SparkJob can be specified with:
+ * <pre>
+ *   def flags = ['version','verbose','help']
+ *   def params = ['param1','param2']
+ *
+ *   sparkJob('jobName') {
+ *     uses 'com.linkedin.azkaban.spark.HelloSpark'  // Required
+ *     executes './lib/azkaban-spark-job.jar' // Required
+ *     appParams params // application parameters
+ *     enableFlags flags
+ *
+ *     set Properties: [
```

Comment:
I'm confused about "set Properties:" - is it supposed to be the same as the regular "set properties:" method? If so, can you fix the documentation here to use the lowercase "properties:" since it's case-sensitive. If it's something different, can we make it more descriptive, like "set sparkProperties:" or something like that?